### PR TITLE
feat: start of nutrition game migration

### PR DESCRIPTION
**Pull request by alexfauquette**
Beggining of the issue #47 

From the reactJs version and the current vueJs code, I implemented the fetching logic, and the display of the table to fill.

Is still missing the upload logic, and the CSS should be improved. If someone wants to continue during the summer feel free

image

### CODE REVIEW

File: src/data/nutritions.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (astagi) [inline src/data/nutritions.json]: @alexfauquette labels should be translatable.

File: src/App.vue [hunk 2]
```diff
@@ -7,6 +7,7 @@
             <sui-dropdown-header>{{$t('menu.games')}}</sui-dropdown-header>
             <router-link class="item" to="/questions">{{$t('menu.questions')}}</router-link>
             <router-link class="item" to="/logos">{{$t('menu.logos')}}</router-link>
+            <router-link class="item" to="/nutritions">{{$t('menu.nutritions')}}</router-link>
```
Comment (teolemon) [inline src/App.vue]: https://github.com/openfoodfacts/openfoodfacts-hungergames/blob/master/src/i18n/common.json
Translation to be added in there

File: src/views/NutritionView.vue [hunk 3]
```diff
@@ -0,0 +1,234 @@
+<template>
+  <div class="ui grid stackable">
+    <div class="five wide column centered">
+      <div class="insight-column">
+        <div class="ui form">
+          <div class="ui icon input" id="value-tag-input">
```
Comment (teolemon) [inline src/views/NutritionView.vue]: the filter can probably be deleted

File: src/views/NutritionView.vue [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+                  </span>
+                </div>
+              </div>
+              <div class="cell ui toggle checkbox">
+                <input
+                  type="checkbox"
+                  name="public"
+                  v-model="currentProductData[nutritiveValue.id]['visible']"
+                />
+                <label>{{ $t("nutrition.Absent") }}</label>
```
Comment (teolemon) [inline src/views/NutritionView.vue]: https://github.com/openfoodfacts/openfoodfacts-hungergames/blob/master/src/i18n/common.json

**Comment by teolemon** [comment]
- [ ] Adding a link to see the product page on the web version
- [ ] Adding an unselect nutrition photo button
- [ ] Adding a way to crop precisely the photo
- [ ] Remove the field to filter (Fanta)
- [ ] Add a save button
- [ ] Add a Skip product button
- [ ] Prepopulate using Robotoff API

File: src/data/nutritions.json (hunk 1 above)
Comment (alexfauquette) [reply src/data/nutritions.json]: I assume `{{ nutritiveValue.name }}` should become ``{{ $t(`nutrition.names.${nutritiveValue.name}`) }}``

Comment (alexfauquette) [reply src/data/nutritions.json]: I add it to the todo list

**Comment by alexfauquette** [comment]
* [x]  Adding a link to see the product page on the web version
 * [ ]  Adding an unselect nutrition photo button
 * [ ]  Adding a way to crop precisely the photo
 * [x]  Remove the field to filter (Fanta)
 * [x]  Add a save button
 * [x]  Add a Skip product button
 * [ ]  Prepopulate using Robotoff API
 * [x] Nutriments name are translatable

I tested two times the API call. Not sure I didn't miss specific cases

**Comment by alexfauquette** [comment]
To prepopulate data with robotoff I encounter the same pb than with react huggergame : 

When I try with examples from the game in local condition, I get images like this
https://static.openfoodfacts.org/images/products/871/120/035/1602/nutrition_fr.5.full.jpg

The corresponding OCR json should be at this adress but I get a 404 error
https://static.openfoodfacts.org/images/products/871/120/035/1602/nutrition_fr.5.json



If I try with a random image in the home page of OFF I get
https://static.openfoodfacts.org/images/products/325/039/121/4365/nutrition_fr.14.full.jpg

which should correspond to the wollowing url (and in this case it's ok)
https://static.openfoodfacts.org/images/products/325/039/121/4365/nutrition_fr.14.json

Do you have an idea of why the OCR json are missing ?

**Comment by alexfauquette** [comment]
In the commit "WIP unselect nutrition images" I add the button to remove an image from selection. But It's refused by the API. Could it be because the request come from my localhost ?

**Pull request merged**